eval/requirements.txt

```diff
@@ -0,0 +1 @@
+kfp[kubernetes]
```

Comment:
I guess this ends up in the `Containerfile` - in the image, right? Why do we need the DSL to be there?

Reply:
I think I remember seeing a "module not found" error without it - but probably you are right, since you didn't need it for the sdg workflow - I've removed it.

Reply:
I'm confused. Did you forget to push the latest changes or you didn't remove it?

Reply:
oh I thought I removed it - 👀 maybe didn't push that.....

Reply:
updated, done